dockstore-webservice/src/main/java/io/dockstore/webservice/resources/ResourceConstants.java

```diff
@@ -31,6 +31,7 @@
     public static final String TOOLTESTER = "Interactions with the Dockstore-support's ToolTester application";
     public static final String ORGANIZATIONS = "Operations on Dockstore organizations";
     public static final String CURATION = "List and modify notifications for users of Dockstore";
+    public static final String NIHDATACOMMONS = "Needed for SmartAPI compatibility apparantly, might be cargo cult behaviour";
```

Comment:
Do you the "cargo cult behavior" in there?

Reply:
?

Reply:
apparantly typo btw

Reply:
I meant, do you mean to leave the "cargo cult behavior" in there? I hadn't heard the expression before, but it sounds vaguely insulting. But maybe it's because I hadn't heard it before.

Reply:
Hah, could be self deprecating I suppose

Reply:
Created https://github.com/dockstore/dockstore/issues/3178 for follow-up